esp-hal/src/uart/mod.rs

```diff
@@ -617,6 +619,7 @@ where
     type ConfigError = ConfigError;
 
     fn set_config(&mut self, config: &Self::Config) -> Result<(), Self::ConfigError> {
+        self.baudrate = config.baudrate;
```

Comment:
Would it be acceptable to not store baud rate, and to modify `fn send_break` to take microseconds (or `esp_hal::time::Duration`, even)?

Reply:
It is possible, but typically breaks are in bit time

Reply:
Okay, nevermind then